.pyre_configuration

```diff
@@ -0,0 +1,17 @@
+{
+  "binary": "/home/johnsonhj/.venv/monai37/bin/pyre.bin",
```

Comment:
please remove this temp test file with your own paths.

Reply:
Thank you for finding this rogue file.  I have removed it from history and added to .gitignore.